### PR TITLE
feat(config): add support for FGWCEU-201

### PR DESCRIPTION
**Pull request by popescunsergiu**

### CODE REVIEW

File: packages/config/config/devices/0x010f/fgwceu-201.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
+{
+	"manufacturer": "Fibargroup",
+	"manufacturerId": "0x010f",
+	"label": "FGWCEU-201",
+	"description": "WALLI FIBARO CONTROLLER",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgwceu-201.json]:
```suggestion
	"description": "Walli Controller",
```

File: packages/config/config/devices/0x010f/fgwceu-201.json [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+		{
+			"productType": "0x2301",
+			"productId": "0x1000"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgwceu-201.json]: We don't use that property anymore. Is that auto-generated or are you adding it?
```suggestion
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Like in the other closed PR, the parameters are similar to the ones in #3169 - please align their definitions so they match, ideally use the same `$import`s where applicable.

File: packages/config/config/devices/0x010f/fgwceu-201.json (hunk 2 above)
Comment (popescunsergiu) [reply packages/config/config/devices/0x010f/fgwceu-201.json]: I was adding it. Will not add it anymore

File: packages/config/config/devices/0x010f/fgwceu-201.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+					"value": 7
+				},
+				{
+					"label": "Blinking red-white-blue",
+					"value": 8
+				}
+			]
+		},
+		"151": {
+			"$import": "#paramInformation/150",
```
Comment (blhoward2) [inline packages/config/config/devices/0x010f/fgwceu-201.json:123]: @AlCalzone Confirm this is the right format for a reference within the same file? I've never actually done that in a device file...

File: packages/config/config/devices/0x010f/fgwceu-201.json [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+			"label": "LED Frame – Second Channel color"
+		},
+		"152": {
+			"label": "First Channel - Double click value",
```
Comment (blhoward2) [inline packages/config/config/devices/0x010f/fgwceu-201.json]: Label should be Title Case. Please capitalize Click and Value

File: packages/config/config/devices/0x010f/fgwceu-201.json [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+			"description": "Defines value of Basic Set or Multilevel Set frame sent to associated devices",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 99,
+			"options": [
+				{
+					"label": "Disabled",
```
Comment (blhoward2) [inline packages/config/config/devices/0x010f/fgwceu-201.json]: Change to "Disable"

File: packages/config/config/devices/0x010f/fgwceu-201.json [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+			"productId": "0x1000"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"13": {
+			"label": "LED Frame – Brightness",
```
Comment (blhoward2) [inline packages/config/config/devices/0x010f/fgwceu-201.json]: To match our files elsewhere, please make this "LED Frame: Brightness". Carry to the others as well.

File: packages/config/config/devices/0x010f/fgwceu-201.json [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 255
+				}
+			]
+		},
+		"153": {
+			"$import": "#paramInformation/152",
+			"label": "Second Chanel - Double click value"
```
Comment (blhoward2) [inline packages/config/config/devices/0x010f/fgwceu-201.json]: Title Case as well, please.

**Review by blhoward2 — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x010f/fgwceu-201.json (hunk 3 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x010f/fgwceu-201.json:123]: yes it is

**Comment by popescunsergiu** [comment]
@AlCalzone  is there anything else I have to do here? thx

**Review by blhoward2 — APPROVED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**